NEWS.md

```diff
@@ -1,3 +1,5 @@
+# phsopendata 1.0.0
+
```

Comment:
Probably add 'Initial CRAN release' or something as a news item. Can't remember if there were other changes (that matter to users) since the last release?